diff --git a/Cargo.toml b/Cargo.toml
index 35f57905..34f26bf0 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -93,7 +93,7 @@ predicates = "3"
 bluer = "0.17"
 hex = "0.4"
 async-trait = "0.1"
-libcosmic = { git = "https://github.com/pop-os/libcosmic.git" }
+libcosmic = { git = "https://github.com/pop-os/libcosmic.git", default-features = false }
 rusqlite = "0.40"
 syn = "3"
 quote = "1"
diff --git a/gui/Cargo.toml b/gui/Cargo.toml
index fb3a13e8..c7ee7752 100644
--- a/gui/Cargo.toml
+++ b/gui/Cargo.toml
@@ -23,6 +23,9 @@ libcosmic = { workspace = true, features = [
     "winit",
     "wgpu",
     "about",
+    "x11",
+    "wayland",
+    "multi-window",
 ] }
 i18n-embed = { workspace = true, features = [
     "fluent-system",
